docs/source/GetStartedGuide.rst

```diff
@@ -180,6 +180,21 @@ system with the right compilers, header files and libraries available.
 The more predictable and complete Avocado experience can be achieved with
 the official RPM packages.
 
+Plugins
```

Comment:
Well for Fedora and RHEL we list these directly in the same section which IMO looks better as the difference between 3rd and 4th heading is so minimal that it almost looks like a new category. Would you please use the same format as used there?

Reply:
I'd mention something like "The optional plugins are also available on pypi", for example...